### ISSUE TITLE
Cannot "click" Dropbox (no response)

### ISSUE THREAD

**Issue by edhilgendorf** [200ok-ch/organice#268]
**Describe the bug**
Hi, I have cloned this project and run without issue using docker or docker-compose, yet cannot use Drive or Dropbox, although the example works great.

**To Reproduce**
Installed on CentOS7, hitting the application directly I get the home screen. Everything in "View Sample" works as expected.
When I reload the page with inspector, I only show 200''s in the network tab.
Clicking Dropbox I do not get any response. 

**Expected behavior**
API Request that I get if I visit https://organice.200ok.ch

**Screenshots**

**Desktop (please complete the following information):**
 - OS: CentOS7
 - Browser: FireFox, Chrome

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy Note9
 - OS: Andoird
 - Browser Firefox,chrome,samsung

Drive also does not work, but I see it hitting the Google API and returning a 403 -- I don't see anything happening in the network tab when I click Dropbox, and have no idea where to start. I get the same for all browsers.
I am not running this on localhost -- It is running on a VM and I'm hitting it directly with the local IP, or my domain: https://org.hilgendorf.me  Thanks so much, I love this project and would love to self-host it successfully!

**Comment by edhilgendorf** [comment]
User error

**Comment by munen** [comment]
:+1: 

All the best using organice :pray:

**Comment by forksnd** [comment]
@edhilgendorf What was the "User error" that you encountered? I am having the same issue, using `docker-compose` and Traefik as a reverse proxy.

**Comment by forksnd** [comment]
Google Drive gives this error:

    TypeError: gapi.auth2.getAuthInstance() is null

Do we need some kind of authentication before deploying the Docker Container? The [documentation](https://organice.200ok.ch/documentation.html#org42caa8d) doesn't mention anything.

**Comment by forksnd** [comment]
The relevant part of the `docker-compose.yml` is 

```
  organice:
    image: "twohundredok/organice:latest"
    container_name: organice
    labels:
      - "traefik.backend=organice"
      - "traefik.docker.network=web"
      - "traefik.enable=true"
      - "traefik.frontend.rule=Host:org.namandixit.net"
      - "traefik.port=5000"
    restart: unless-stopped
    networks:
      - web
```

**Comment by forksnd** [comment]
The relevant part of `docker-compose.yml` is:

````
  organice:
    image: "twohundredok/organice:latest"
    container_name: organice
    labels:
      - "traefik.backend=organice"
      - "traefik.docker.network=web"
      - "traefik.enable=true"
      - "traefik.frontend.rule=Host:org.example.com"
      - "traefik.port=5000"
    restart: unless-stopped
    networks:
      - web
````

**Comment by munen** [comment]
@forksnd You need to supply the proprietary back-ends with the respective credentials. Documentation is here: https://organice.200ok.ch/documentation.html#synchronization_back_ends

All the best using organice pray

**Comment by forksnd** [comment]
This is what I did:

1. Went to the Dropbox page you linked to 
2. Created an app with redirect URL `org.example.com` (example.com replaced with actual domain)
2. In `docker-compose.yml`, set the value of `REACT_APP_DROPBOX_CLIENT_ID` to "App Key" the first time, and "App Secret" the second
3. Tried logging in.

At this point, both times, I recieved the same error. Is there some other variable I need to set? The documentation that you linked to doesn't contain a lot of detail.

Thanks.
